src/org/usfirst/frc/team1076/robot/commands/DoorMovement.java

```diff
@@ -0,0 +1,49 @@
+package org.usfirst.frc.team1076.robot.commands;
+
+import org.usfirst.frc.team1076.robot.subsystems.DoorPneumatic;
+
+import edu.wpi.first.wpilibj.Solenoid;
+import edu.wpi.first.wpilibj.command.Command;
+
+/**
+ * Controls whether or not to open or close the door.
+ */
+public class DoorMovement extends Command {
+
+	DoorPneumatic doormovement; 
```

Comment:
This should have a camelCase name like `doorMovement`, although I think a more descriptive name like `doorPiston` would be better.